esphome/components/chenyang/cover.py

```diff
@@ -0,0 +1,42 @@
+import esphome.codegen as cg
+import esphome.config_validation as cv
+from esphome.components import cover, rs485
+from esphome.const import CONF_ADDRESS, CONF_ID, CONF_UPDATE_INTERVAL
+
+CODEOWNERS = ["@loongyh"]
+
+AUTO_LOAD = ["rs485"]
+
+chenyang_ns = cg.esphome_ns.namespace("chenyang")
+Chenyang = chenyang_ns.class_(
+    "Chenyang", cover.Cover, cg.PollingComponent, rs485.RS485Device
+)
+
+CONFIG_SCHEMA = (
+    cover.COVER_SCHEMA.extend(
+        {
+            cv.GenerateID(): cv.declare_id(Chenyang),
+            cv.Optional(CONF_ADDRESS): cv.hex_uint8_t,
+            cv.Optional(
+                CONF_UPDATE_INTERVAL, default="1s"
+            ): cv.positive_time_period_milliseconds,
+        }
+    )
+    .extend(cv.COMPONENT_SCHEMA)
```

Comment:
Instead of setting `update_interval` above, write:
```suggestion
    .extend(cv.polling_component_schema("1s"))
```

Reply:
Isn't this hardcoding the update interval? Will the user still be able to configure it via `update_interval:` in .yaml?

Reply:
`def polling_component_schema(default_update_interval):`

oh, just saw config_validation.py, yep it's still configurable: `Optional(CONF_UPDATE_INTERVAL, default=default_update_interval): update_interval,`